cfgmgr/teammgr.cpp

```diff
@@ -654,10 +656,46 @@ bool TeamMgr::removeLag(const string &alias)
 
     stringstream cmd;
     string res;
+    pid_t pid;
 
     cmd << TEAMD_CMD << " -k -t " << shellquote(alias);
     EXEC_WITH_ERROR_THROW(cmd.str(), res);
```

Comment:
This could be causing issues. IIUC if the process actually wasn't running, then `teamd -k -t PortChannel.1` would fail. So in the case where `addLag()` fails, but no `teamd ...` running, then `teamd -k -t PortChannel.` would fail, as a result an exception will be thrown, could you check?